db/migrate/20240917080733_create_patient_matches.rb

```diff
@@ -0,0 +1,23 @@
+class CreatePatientMatches < ActiveRecord::Migration[7.0]
+  def change
+    create_table :patient_matches do |t|
+      t.integer :patient_id_a, null: false, index: true   # Primary patient ID
+      t.integer :patient_id_b, null: false, index: true   # Secondary (duplicate) patient ID
+      t.integer :match_percentage                        
+      t.boolean :merge_status,null: false, default: false
+      
+      t.datetime :date_created
```

Comment:
Aren't these duplicating the t.timestamps?

Reply:
i have made the changes